05-single-hop-route/test_spec05.py

```diff
@@ -0,0 +1,131 @@
+import time
+
+import pytest
+
+from riotctrl_shell.gnrc import GNRCICMPv6Echo, GNRCIPv6NIB, GNRCPktbufStats
+from riotctrl_shell.netif import Ifconfig
+
+from testutils.native import bridged
+from testutils.shell import ping6, pktbuf, lladdr
+
+
+APP = 'examples/gnrc_networking'
+pytestmark = pytest.mark.rc_only()
+
+
+class Shell(Ifconfig, GNRCICMPv6Echo, GNRCPktbufStats, GNRCIPv6NIB):
+    pass
+
+
+@pytest.mark.skipif(not bridged(["tap0", "tap1"]),
+                    reason="tap0 and tap1 not bridged")
+# nodes passed to riot_ctrl fixture
+@pytest.mark.parametrize('nodes',
+                         [pytest.param(['native', 'native'])],
+                         indirect=['nodes'])
+def test_task01(riot_ctrl):
+    pinger, pinged = (
+        riot_ctrl(0, APP, Shell, modules=["gnrc_pktbuf_cmd"], port="tap0"),
+        riot_ctrl(1, APP, Shell, modules=["gnrc_pktbuf_cmd"], port="tap1"),
+    )
+
+    pinged_netif, pinged_lladdr = lladdr(pinged.ifconfig_list())
+    pinged.ifconfig_flag(pinged_netif, "rtr_adv", False)
+    pinged.ifconfig_add(pinged_netif, "beef::1/64")
+    pinger_netif, pinger_lladdr = lladdr(pinger.ifconfig_list())
+    pinger.ifconfig_flag(pinger_netif, "rtr_adv", False)
+    pinger.ifconfig_add(pinger_netif, "beef::2/64")
+
+    pinged.nib_route_add(pinged_netif, "::", pinger_lladdr)
+    pinger.nib_route_add(pinger_netif, "::", pinged_lladdr)
+
+    res = ping6(pinger, "beef::2",
```

Comment:
```suggestion
    res = ping6(pinger, "beef::1",
```

By the spec this should be the other way around, right now the node is pinging itself.

Reply:
Oops. Fixed!